sentence_transformers/SentenceTransformer.py

```diff
@@ -465,8 +465,8 @@ def save_to_hub(self,
 
         endpoint = "https://huggingface.co"
         repo_url = HfApi(endpoint=endpoint).create_repo(
-                token,
-                repo_name,
+                token=token,
+                repo_id=repo_name,
```

Comment:
The `organization` parameter is deprecated and will soon be removed in next release of `huggingface_hub` (a Warning message should already be printed since version `0.8`).

Suggested change is to add organization to the repo id and remove it from the arguments passed to `create_repo`.

```suggestion
                repo_id=f"{organization}/{repo_name}",
```

Reply:
Organization might be `None` though and that would lead to an error I think.

Maybe we should go ahead with https://github.com/UKPLab/sentence-transformers/pull/1743 ?

Reply:
Yes sorry, my bad. Checking this would be better :)